app/views/admin/needs/edit.html.erb

```diff
@@ -1,9 +1,6 @@
 <% content_for :page_title, @edition.title %>
 <% content_for :title, "Associated user needs" %>
 <% content_for :context, @edition.title %>
-<% content_for :back_link do %>
-  <%= render "govuk_publishing_components/components/back_link", href: admin_edition_path(@edition) %>
```

Comment:
Do we need to add a cancel button here?

Reply:
The bootstrap page doesn't have a cancel button, JP agreed that with like for like we shouldn't introduce one if there wasn't one there before.